play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukSummaryList.scala.html

```diff
@@ -90,7 +91,6 @@
 @actionLink(action: ActionItem, cardTitle: Option[Content]) = {
     <a class="@toClasses("govuk-link", action.classes)" href="@action.href"@toAttributes(action.attributes)>@action.content.asHtml@visuallyHiddenText(action, cardTitle)</a>}
 
-@visuallyHiddenText(action: ActionItem, cardTitle: Option[Content]) = {@if(action.visuallyHiddenText.exists(_.nonEmpty) || cardTitle.exists(_.nonEmpty)) {<span class="govuk-visually-hidden">
-    @{action.visuallyHiddenText}@{cardTitle.filter(_.nonEmpty).map(ct => s" (${ct.asHtml})")}</span>}}
+@visuallyHiddenText(action: ActionItem, cardTitle: Option[Content]) = {@if(action.visuallyHiddenText.exists(_.nonEmpty) || cardTitle.isDefined) {<span class="govuk-visually-hidden"> @{action.visuallyHiddenText.map(_.rtrim)}@if(cardTitle.isDefined){@cardTitle.map {content => (@content.asHtml)}}</span>}}
```

Comment:
I see this has changed from `cardTitle.exists(_.nonEmpty)` to `cardTitle.isDefined` - is that correct? The first checks that the cardTitle is Some AND the string is non-empty, the new version will just check for Some and not None.

Reply:
So following the nunjucks example and trying to keep as close to it as possible in our twirl templates. The cardTitle can be an empty object which causes that check to fail when its `cardTitle.exists(_.nonEmpty)` and therefore produces the incorrect output. We use `cardTitle.isDefined` to ensure we produce the same output as the nunjucks examples.